client/fire-webclient/src/main/kotlin/uk/ac/ebi/fire/client/integration/web/RetryWebClient.kt

```diff
@@ -25,11 +26,6 @@ internal class RetryWebClient(
         template.execute(opt) { fireClient.unsetPath(fireOid) }
     }
 
-    override fun downloadByPath(path: String): File? {
-        val opt = "Download file path='$path'"
-        return template.execute(opt) { fireClient.downloadByPath(path) }
```

Comment:
Since we're removing this, I think we should also remove the [http client method](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/client/fire-webclient/src/main/kotlin/uk/ac/ebi/fire/client/api/FireWebClient.kt#L49) and the related tests since it won't be required anymore